src/translib/sys_app.go

```diff
@@ -45,6 +45,7 @@ type SysApp struct {
 
 func init() {
     log.Info("SysApp: Init called for System module")
+	return
```

Comment:
can we remove the sys_app.go since your pull request replaces it?

Reply:
Done